autogen/agentchat/chat.py

```diff
@@ -165,10 +165,13 @@ def initiate_chats(chat_queue: List[Dict[str, Any]]) -> List[ChatResult]:
                Default is {}.
             - `"message"` (str, callable or None) - if None, input() will be called to get the
                initial message.
-            - `**context` - additional context information to be passed to the chat.
-            - `"carryover"` - It can be used to specify the carryover information to be passed
+            - `"carryover"` (list) - It can be used to specify the carryover information to be passed
                to this chat. If provided, we will combine this carryover with the "message" content when
                generating the initial chat message in `generate_init_message`.
+            - `"finished_chat_summary"` (bool) - It can be used to specify if past chat summary needs to be passed or not.
```

Comment:
Is "finished_chat_summary" really necessary? The information could presumably be reflected by the content of "carryover_indexes". Including a particular index in "carryover_indexes" means we want to pass it, otherwise not.

Reply:
Yes I think this would be better in my opinion.

Reply:
this has been updated please check.